core/src/stored/backends/win32_tape_device.cc

```diff
@@ -105,20 +105,20 @@ static inline ULONG Read32BitUnsigned(const unsigned char* pValue)
 
 static inline LONGLONG Read64BitSigned(const unsigned char* pValue)
 {
-  return (LONGLONG)(
-      ((ULONGLONG)pValue[0] << 56) | ((ULONGLONG)pValue[1] << 48)
-      | ((ULONGLONG)pValue[2] << 40) | ((ULONGLONG)pValue[3] << 32)
-      | ((ULONGLONG)pValue[4] << 24) | ((ULONGLONG)pValue[5] << 16)
-      | ((ULONGLONG)pValue[6] << 8) | (ULONGLONG)pValue[7]);
+  return (
+      LONGLONG)(((ULONGLONG)pValue[0] << 56) | ((ULONGLONG)pValue[1] << 48)
+                | ((ULONGLONG)pValue[2] << 40) | ((ULONGLONG)pValue[3] << 32)
+                | ((ULONGLONG)pValue[4] << 24) | ((ULONGLONG)pValue[5] << 16)
+                | ((ULONGLONG)pValue[6] << 8) | (ULONGLONG)pValue[7]);
```

Comment:
Splitting up the cast looks weird, but i guess this was done by bareos-check-sources so its fine.